src/app/body/milestone-details/milestone-details.component.css

```diff
@@ -115,7 +116,7 @@
 #progress-bar-container #line {
     width: 100%;
     margin: auto;
-    background: #eee;
+    background: rgb(169, 169, 169);
```

Comment:
Color can be fixed with code and better use theme color

Reply:
yes sir